### DISCUSSION TITLE
有个楼盘发布了待停贷消息，不会添加，谁帮忙添加一下

### DISCUSSION THREAD

**Discussion by FeelBreeZe** [WeNeedHome/SummaryOfLoanSuspension#1134] [General]
河南鹤壁市淇县建业城
![IMG_4854_20220817-195141_.JPG](https://s2.loli.net/2022/08/17/vZ5QJqmHFKkOMeh.jpg)

**Comment by yuhan6665** [comment]
最近管理员比较忙 有18个pr在等。。 之后我会帮你加上的

**Reply by FeelBreeZe** [reply]
谢谢

**Reply by ewkyrtas** [reply]
> 谢谢

**### 是不是 管理被请去喝茶了都8天没有更新了   不是有很多管理员的吗？？？？？？？？？？？？？？？？？？？？？？？**

**Comment by Cegazhos** [comment]
顶顶，话说不能下月就停么 还要等到12月才停

**Reply by JayChou55** [reply]
奴性十足。。。唯唯诺诺

**Reply by ewkyrtas** [reply]
是不是 管理被请去喝茶了都8天没有更新了 不是有很多管理员的吗？？？？？？？？？？？？？？？？？？？？？？？

**Reply by Ventery** [reply]
> 奴性十足。。。唯唯诺诺

傻逼东西，闭嘴吧

**Reply by JayChou55** [reply]
> > 奴性十足。。。唯唯诺诺
> 
> 傻逼东西，闭嘴吧

戳到你这狗b痛点了？

**Reply by Cegazhos** [reply]
> 奴性十足。。。唯唯诺诺

傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？

**Reply by JayChou55** [reply]
> > 奴性十足。。。唯唯诺诺
> 
> 傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？

说点实话就给人扣台湾帽，你顶着日本游戏的头像，你狗日的 是日本人艹出的杂种？粉蛆是真尼玛的 弱智 再回我 你全家暴bi 母亲得癌 子女白血病

**Reply by JayChou55** [reply]
> 

狗B东西，玩还拿日本游戏当头像 真的

> > 奴性十足。。。唯唯诺诺
> 
> 傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？

你jb真有种，就别tm来github 你美帝的东西 你用 跟tm真的一样 跟着国家走 你tm 抵制啊 傻狗。。

**Reply by Cegazhos** [reply]
> > > 奴性十足。。。唯唯诺诺
> > 
> > 
> > 傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？
> 
> 说点实话就给人扣台湾帽，你顶着日本游戏的头像，你狗日的 是日本人艹出的杂种？粉蛆是真尼玛的 弱智 再回我 你全家暴bi 母亲得癌 子女白血病

你个傻卵子 对号入座还说自己不是台湾狗？国语课没上过还是没读书啊？

**Reply by Cegazhos** [reply]
> > 
> 
> 狗B东西，玩还拿日本游戏当头像 真的
> 
> > > 奴性十足。。。唯唯诺诺
> > 
> > 
> > 傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？
> 
> 你jb真有种，就别tm来github 你美帝的东西 你用 跟tm真的一样 跟着国家走 你tm 抵制啊 傻狗。。

自己像条野狗一样到处乱喷粪 乱扣帽子

> > > 奴性十足。。。唯唯诺诺
> > 
> > 
> > 傻逼台湾狗 人家又不是在暴扣你妈的臭批 怎么啥事都来尿一下？
> 
> 说点实话就给人扣台湾帽，你顶着日本游戏的头像，你狗日的 是日本人艹出的杂种？粉蛆是真尼玛的 弱智 再回我 你全家暴bi 母亲得癌 子女白血病

自己像条野狗一样先到处乱喷粪 乱扣帽子 还要别人不许骂回你 要哭哭惹 真是有够幼稚的。
还挺有意思的吖，霓虹人不是你台湾傻狗的爹么？在岛上招商投资给你们这群废物吃饭，你个软狗蛋子这会出来划清关系了？
笑死🤣

**Reply by JayChou55** [reply]
粉红是这个世界最可怜的东西。。。你自己慢慢嗨，哥今年就润了

**Reply by Cegazhos** [reply]
> 粉红是这个世界最可怜的东西。。。你自己慢慢嗨，哥今年就润了

你死哪都不关我事啊，我既不是你爹，也不是你妈的粉批。跟个傻狗一样到处喷粪 扣帽子，然后自我高潮，这就是台湾乡巴佬么

**Reply by fnnnnng** [reply]
看这措辞的确是纯种粉红，别生气了，你主人发个三好奴才证给你贴墙上

**Reply by Cegazhos** [reply]
> 看这措辞的确是纯种粉红，别生气了，你主人发个三好奴才证给你贴墙上

共党是你爹？看来你妈的批还不够黑，以致于你这种傻卵看什么都是粉的

**Comment by wanlulu920305** [comment]
> 河南鹤壁市淇县建业城
> 
> ![IMG_4854_20220817-195141_.JPG](https://s2.loli.net/2022/08/17/vZ5QJqmHFKkOMeh.jpg)
> 
> 

你是淇县的吗

**Reply by Kylin2021** [reply]
襄阳融创民发御湖壹号 也烂尾了 已经开始在门口拉横幅了

**Comment by Kylin2021** [comment]
襄阳融创民发御湖壹号 也烂尾了 已经开始在门口拉横幅了

**Reply by lettermessenger** [reply]
挂横幅是违法的 ，要挨打的

**Reply by Cegazhos** [reply]
> 襄阳融创民发御湖壹号 也烂尾了 已经开始在门口拉横幅了

有停贷书可以发出来
